### ISSUE TITLE
Add test for markChapterCompleted in StateFragment [Blocked on #495]

### ISSUE THREAD

**Issue by rt4914** [oppia/oppia-android#880]
Once #495 is merged into develop, write test cases for `Return to Topic` functionality in app module code. This will be done later because in current develop test cases in player package are failing but updating those test cases is not a good idea as they are getting updating in #495 along with entire implementation change.

**Comment by rt4914** [comment]
This was finished in #495 . Reference: https://github.com/oppia/oppia-android/blob/5fc69fffb97e0c8a0d529238926f3e078cf88b09/app/src/sharedTest/java/org/oppia/android/app/player/state/StateFragmentTest.kt#L830
